scripts/jmx4perl

```diff
@@ -237,9 +236,8 @@ documentation abouting listing of MBeans.
 =cut
 
 sub command_list {
-    my ($jmx,$path) = @_;
-    $path =~ s|:|/|;
-    my $req = JMX::Jmx4Perl::Request->new(LIST,$path);
+    my ($jmx,$mbean,$path) = @_;
+    my $req = JMX::Jmx4Perl::Request->new(LIST,$mbean,$path);
```

Comment:
That won't work as the constructor will pickup only the first argument after LIST as "path" value.

Reply:
The code here needs to be updated, too:

https://github.com/rhuss/jmx4perl/blob/7b75695e6e0bc90b09d8a519141dbfeb7b81ae1c/lib/JMX/Jmx4Perl/Request.pm#L280-L282

Or, even better, use the named approach here (but still update the constructor)

```perl
new JMX::Jmx4Perl::Request(LIST,{ mbean => $mbean, path => $path })
```

Also an update of the `new()` perldoc in `Request` would be awesome.

Reply:
I already did modify JMX::Jmx4Perl::Request to use that second parameter.

And I also already did modify the documentation.